dace/frontend/fortran/ast_transforms.py

```diff
@@ -2239,7 +2239,7 @@ def par_Decl_Range_Finder(node: ast_internal_classes.Array_Subscript_Node,
                 newbody.append(
                     ast_internal_classes.Decl_Stmt_Node(vardecl=[
                         ast_internal_classes.Symbol_Decl_Node(
-                            name="tmp_parfor_" + str(count + len(rangepos) - 1), type="INTEGER", sizes=None, init=None,
```

Comment:
parDeclRangeFinder is called from many different transformations. Do you think this can still cause name collisions? if yes, then we can change it by passing a custom prefix from each transform